### DISCUSSION TITLE
How do we change graph to 300mm?

### DISCUSSION THREAD

**Discussion by Ltek** [jneilliii/OctoPrint-BedLevelVisualizer#542] [Q&A]
With the visual diagram, it always shows 200mm ... my Sidewinder X2 is 300mm .. any way to change the diagram to be accurate?

thx

**Comment by jneilliii** [comment]
the bed diameter is based on what you have configured in OctoPrint's printer profile settings.

**Reply by Ltek** [reply]
thank you... I didnt realize I forgot to update Octo settings!
